client/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use serde::de::DeserializeOwned;
+use serde::Deserialize;
+use std::net::SocketAddr;
+use std::str::FromStr;
+
+pub(crate) const DEFAULT_NODE_ADDR: &str = "127.0.0.1:3030";
+pub(crate) const DEFAULT_PROTOCOL: &str = "http";
+
+/// The Client Node Configuration. Should be loaded from a static file.
+#[derive(Deserialize, Debug, Default)]
+pub struct ClientNodeConfig {
```

Comment:
I think there should be a single config for the whole IPC client/node that is shared. We should think as the `IPC client` as a single application. Actually, calling it client may not be the best naming, and we should maybe call it `ipc-node`.